### PR TITLE
BUG - Fix image tag format

### PR DESCRIPTION
**Pull request by trallard**
- :memo: Add links to all the images
- :bug: Patch image tag

## What does this implement/fix?

_Put a `x` in the boxes that apply_

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds a feature)
- [ ] Breaking change (fix or feature that would cause existing features to not work as expected)
- [ ] Documentation Update
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no API changes)
- [ ] Build related changes
- [ ] Other (please describe):

## Testing

- [x] Did you test the pull request locally?
- [ ] Did you add new tests?

#### Text styling

- [ ] The content is written with [plain language](https://www.plainlanguage.gov/guidelines/) (where relevant).
- [ ] If there are headers, they use the proper header tags (with only one level-one header: `H1` or `#` in markdown).
- [ ] All links describe where they link to (for example, check the [Nebari website](https://nebari.dev/)).
- [ ] This content adheres to the Nebari style guides.

#### Non-text content

- [ ] All content is represented as text (for example, images need alt text, and videos need captions or descriptive transcripts).
- [ ] If there are emojis, there are not more than three in a row.
- [ ] Don't use [flashing GIFs or videos](https://www.w3.org/TR/UNDERSTANDING-WCAG20/seizure-does-not-violate.html).
- [ ] If the content were to be read as plain text, it still makes sense, and no information is missing.

### CODE REVIEW

File: .github/workflows/build-push-docker.yaml [hunk 1]
```diff
@@ -93,7 +93,7 @@ jobs:
             "ghcr.io/${{ github.repository_owner }}/nebari-${{ matrix.dockerfile }}${{ env.IMAGE_SUFFIX }}"
           tags: |
             # branch event -> e.g. `main-f0f6994-20221001`
-            type=ref,event=branch, suffix={{sha}}-{{date 'YYYYMMDD'}}
+            type=ref,event=branch, suffix=-{{sha}}-{{date 'YYYYMMDD'}}
```
Comment (iameskild) [inline .github/workflows/build-push-docker.yaml:96]: We have been using the `main` tag for testing in other repos (kubernetes tests, integration tests, etc) but the addition of this suffix will break those. If we want both, can we also add a tag for the branch name?

**Review by iameskild — CHANGES_REQUESTED** [review]

File: .github/workflows/build-push-docker.yaml (hunk 1 above)
Comment (iameskild) [reply .github/workflows/build-push-docker.yaml:96]: @trallard

Comment (trallard) [reply .github/workflows/build-push-docker.yaml:96]: well - last time we discussed the tagging format for the Docker images this is what we agreed on (also why I was asking where all the Docker images needed updating)
either we add another tag here or fix elsewhere

Comment (iameskild) [reply .github/workflows/build-push-docker.yaml:96]: @trallard in my defense, I recall having this conversation and I also recall pushing back for the reason stated above. I just realized that this change will also break how we initialize the default image tags in the `nebari-config.yaml::default_images` section.

**Pull request merged**